contributing/code/standards.rst

```diff
@@ -167,6 +167,23 @@ Service Naming Conventions
 
 * A group name uses the underscore notation.
 
+.. _contributing-code-standards-comparisons:
+
+Comparisons
+-----------
+
+Always use `identical comparison`_ unless you need type juggling::
```

Comment:
You mean in [structure](https://symfony.com/doc/current/contributing/code/standards.html#structure) section?